api/src/main/java/de/valtech/aecu/api/service/AecuService.java

```diff
@@ -44,10 +44,22 @@
 public interface AecuService {
 
     /**
-     * Prefix in CRX where AECU scripts are located.
+     * Prefix in repository where AECU scripts are located.
+     * @deprecated Use either {@link #AECU_VAR_PATH_PREFIX} or {@link #AECU_CONTENT_PATH_PREFIX}
      */
-    public static final String AECU_PREFIX = "/var/groovyconsole/scripts/aecu";
+    @Deprecated
+    public static final String AECU_PREFIX = "/etc/groovyconsole/scripts/aecu";
 
+    /**
+     * Prefix in repository where ad-hoc AECU scripts are located.
+     */
+    public static final String AECU_VAR_PATH_PREFIX = "/var/groovyconsole/scripts/aecu";
+
+    /**
+     * Prefix in repository where install hook AECU scripts are located.
+     */
+    public static final String AECU_CONTENT_PATH_PREFIX = "/content/groovyconsole/scripts/aecu";
```

Comment:
What about /conf/groovyconsole/scripts/aecu? Wouldn't this be a better location? Content folder seems a bit odd to store scripts.

Reply:
Yes, probably better.